cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -43,9 +43,8 @@ import (
 	cloudcontrollerconfig "k8s.io/kubernetes/cmd/cloud-controller-manager/app/config"
 	cmoptions "k8s.io/kubernetes/cmd/controller-manager/app/options"
 	"k8s.io/kubernetes/pkg/controller"
-
-	// add the kubernetes feature gates
-	_ "k8s.io/kubernetes/pkg/features"
+	//// add the kubernetes feature gates
+	//_ "k8s.io/kubernetes/pkg/features"
```

Comment:
Replace these lines with a TODO and bug # for setting up only the public feature gates.